CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
Need to delete the “Unreleased” line for release, then add it back again for the next feature

Reply:
ooo i thought we kept it will do